packages/ibm-products/src/components/Datagrid/Datagrid/DatagridToolbar.js

```diff
@@ -116,7 +110,7 @@ const DatagridBatchActionsToolbar = (datagridState, width, ref) => {
               return null;
             }
             return (
-              <ButtonMenuItem
+              <MenuItem
                 key={`${batchAction.label}-${index}`}
                 itemText={batchAction.label}
                 onClick={(event) => {
```

Comment:
```suggestion
              <MenuItem
                key={`${batchAction.label}-${index}`}
                label={batchAction.label}
                onClick={(event) => {
```

I think this is what’s causing the menu items not to render correctly.

![image](https://github.com/carbon-design-system/ibm-products/assets/1717398/2c41f965-021c-4e78-976c-7107a3c9c340)